BasicSample/Assets/ARMesh/Scripts/MeshFollowUser.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using UnityEngine.XR.ARFoundation;
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    /// <summary>
+    /// Updates the mesh bounding volume every frame to follow the user's head.
+    /// </summary>
+    public class MeshFollowUser : MonoBehaviour
+    {
+        [SerializeField]
+        private ARMeshManager meshManager = null;
+
+        [SerializeField]
+        private Vector3 boundingExtents = Vector3.one * 3;
+
+        private void Update()
+        {
+            meshManager.subsystem.SetBoundingVolume(Camera.main.transform.position, boundingExtents);
```

Comment:
Also consider to add a logic to the "meshManager" here.   If it's null, look for the ARMeshManager on the current game object.  So user can easily drop this to the same game object and get it working.

Reply:
Added!